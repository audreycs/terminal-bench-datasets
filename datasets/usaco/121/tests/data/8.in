110001000010001001100100100000100100110101001010011101001001111111111100111010010000010110001111010101101111110101110100101001101110010110100000011010111001111010100100011101101101100001010111000110110100111000000101010011100011001010111110110100101000100100101111010010100110001100000010000001011000010010001110101001111110111000110110111111000001000000000001110011000110011101101101010111101101100001110100011111001000010010110100001100101110011110010001110010001111110110110100011000011001100111011111010011101000001111100001011001010110000110100000001001011000000011000011101011100001000000010100000010001101100010100001100000101001111001110010110100010111001000011001011010000010100000101010100010111100010011111111000111101010100100111000101010010000100011010101101111001010100001101010100000010001010011101111011110001111011111011100100001001110010110110111101000110011001110010011000101011011111100110010011101
