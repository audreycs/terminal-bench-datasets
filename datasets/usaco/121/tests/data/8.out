1101000001100100100010101100101001110010000111110001101111101001111111001011011110010101111010000100110001101101010010111111000101010011111110100111001001011000100011101011111001000101110111001000110100000010111001011010001100010101111010101011111110110001110000100011111100001001001100100010010111011100110101111001001001101101000110100110101111010001000000011110100100101101111001000011010011000101111110111100010001001100111111110111011000010110000010101110010110001101100011111010011110110011011111010100001101101100000111110111101110110111101110100010011111011000110011111110100011110001000101010100100101100110001010111001101011001000010110011111111010001001001110101111111010101010101011010011010010000001010011110001000010010011110011000011001110011001011000110001100001110010111100010010100100100110001111100111000010000111010110100100110100110100000100110001110101100110110011000100011100011011001001011001101101
